lib/cacheable/response_cache_handler.rb

```diff
@@ -151,8 +159,7 @@ def page_too_old(timestamp, cache_age_tolerance)
 
     def cache_return!(message, &block)
       Cacheable.log message
-      @controller.instance_eval(&block)
-      throw :cache_return
+      throw :cache_return, @controller.instance_eval(&block)
```

Comment:
What is this change for?

Reply:
This is a bit confusing but the second arg to throw is returned from the catch block.